src/gui_state.cpp

```diff
@@ -191,6 +196,14 @@ namespace {
       ImGui::EndMenu();
     }
     ImGui::EndMainMenuBar();
+    #ifdef __APPLE__
```

Comment:
That is not a bad idea, but my variadic template foo couldn't solve the compilation problems I get with the above snippet.

`ImGui::MenuItem` has a field for the key binding, but it does not bind anything as of yet, but has an issue: https://github.com/ocornut/imgui/issues/456

Reply:
That issue has been around since 2015. I do not assume I can solve it in 5 lines I typed out in the browser.

On the compilation issues, the principle is sound, ie https://gcc.godbolt.org/z/ncfnT7